tbx/static_src/sass/components/_blockquote.scss

```diff
@@ -0,0 +1,47 @@
+@use "config" as *;
+
+.blockquote {
+    font-style: normal;
+
+    &__icon {
+        margin-bottom: $spacer-small;
+    }
+
+    &__text {
+        font-style: normal;
+        margin-bottom: $spacer-small;
+
+        @include media-query(medium) {
+            margin-bottom: $spacer-medium;
+        }
+    }
+
+    &__reference {
+        @include media-query(large) {
+            display: flex;
+            gap: 20px;
+            align-items: center;
+            justify-content: space-between;
+        }
+    }
+
+    &__meta {
+        display: flex;
+        align-items: center;
+        gap: 20px;
+    }
+
+    &__cite {
+        font-style: normal;
+    }
+
+    &__name {
+        color: var(--color--heading);
+        display: block;
+        font-weight: $weight--semibold;
+    }
+
+    &__company-logo {
+        filter: grayscale(1);
```

Comment:
Is there a better way of making an image black and white?

Reply:
this was the method I was assuming we'd use.